context/parseLambda.ts

```diff
@@ -270,7 +270,9 @@ export const parseLambda = (
       mutez,
       data:
         lambdaType === LambdaType.CONTRACT_EXECUTION
-          ? JSON.stringify(data.args?.[1], null, 2) ?? {}
+          ? !!data.args?.[1]
+            ? emitMicheline(data.args[1])
+            : `{ "prim": "Unit" }`
```

Comment:
```suggestion
"Unit"
```
it's "Unit". my bad...